### ISSUE TITLE
[Accessibility]: Extension action items on file page lose focus on enter 

### ISSUE THREAD

**Issue by tjkandala** [sourcegraph/sourcegraph#36850]
### Audit type

Keyboard navigation

### User journey audit issue

#34056

### Problem description

On the file page ([example](https://sourcegraph.com/github.com/sourcegraph/go-diff@9d1f353a285b3094bc33bdae277a19aedabe8b71/-/blob/cmd/go-diff/go-diff.go)) with the default extension `git-extras` enabled, pressing `Enter` when the `git-extras` action item is focused will result in loss of focus, even after loading state is done.

### Expected behavior

The action item should retain focus.

### Additional details

https://user-images.githubusercontent.com/37420160/172704698-1fde2d65-c3be-42a6-a7a8-bfbb911256e4.mov

### Assigning labels

- [X] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [X] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [X] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.

**Comment by sourcegraph-bot-2** [comment]
Heads up @muratsu @jjinnii @ryankscott - the "team/integrations" label was applied to this issue.
